web/ui_utils.js

```diff
@@ -189,7 +189,10 @@ function watchScroll(viewAreaElement, callback) {
   };
 
   let rAF = null;
-  viewAreaElement.addEventListener("scroll", debounceScroll, true);
+  viewAreaElement.addEventListener("scroll", debounceScroll, {
+    useCapture: true,
+    signal: abortController?.signal,
```

Comment:
(With the comment above implemented.)
```suggestion
    signal,
```